src/components/ProgressForm/ProgressLayout.tsx

```diff
@@ -8,19 +8,31 @@ import ProgressForm from './ProgressForm';
 import styles from '@/components/ProgressForm/ProgressForm.module.scss';
 
 import { questions } from '@/constants/ProgressUpdates';
+import { getTotalMissedUpdates } from '@/utils/getTotalMissedUpdate';
+import { useUserProgressDetailsQuery } from '@/app/services/progressesApi';
+import { useGetUserQuery } from '@/app/services/userApi';
 
-function ProgressLayout() {
+const ProgressLayout: FC = () => {
+    const { data: user } = useGetUserQuery();
+    const { data: userStandupdata } = useUserProgressDetailsQuery(user?.id);
+    const standupDates = userStandupdata?.data?.map((element) => element.date);
+    const totalMissedUpdates = getTotalMissedUpdates(standupDates || []);
     return (
         <>
             <NavBar />
-            <ProgressHeader />
+            <div className={styles.banner}>
```

Comment:
Is this div used just for styling? Can you move styling into the progressHeader component, please?

Reply:
I will handle this in next issue - https://github.com/Real-Dev-Squad/website-status/issues/638